src/lib/docker/dockerService.ts

```diff
@@ -347,6 +349,24 @@ class DockerService implements DockerLibrary {
       }
     }
   }
+
+  /**
+   * Renames the docker volume directory on disk when a node is renamed
+   * @param network the network containing the node
+   * @param node the node that's to be renamed
+   * @param newName the new name for the node and directory
+   */
+  async updateDirs(network: Network, node: CommonNode, newName: string) {
```

Comment:
Since this function is public, I wouldn't place it down here with the `private` functions. I'd put it under `removeNode` with the other functions that operate on a node. I'd also change the name to `renameNodeDir`.